en/02_Developer_Guides/01_Templates/09_Casting.md

```diff
@@ -6,13 +6,17 @@ icon: code
 
 # Formatting, casting, and escaping variable content
 
-All objects that are being rendered in a template should be a [ModelData](api:SilverStripe\Model\ModelData) instance such as `DataObject`,
-`DBField` or `Controller`. From these objects, the template can include any method from the object in [scope](syntax#scope).
+Classes that subclass [`ModelData`](api:SilverStripe\Model\ModelData) (such as `DataObject`,
+`DBField` or `Controller`) can define how their values should be cast or escaped when used in a template.
```

Comment:
You can pass basically anything into templates, doesn't have to be `ModelData` - but the ability to define casting like this _is_ unique to `ModelData` and its subclasses.